src/main/java/com/redhat/developers/msa/namaste/NamasteApplication.java

```diff
@@ -70,8 +70,12 @@ public void run(NamasteConfiguration configuration, Environment environment) thr
         environment.getApplicationContext().addServlet(HystrixMetricsStreamServlet.class, "/hystrix.stream");
 
         // Register BraveServletFilter
+    	String zipkinUrl = System.getenv("ZIPKIN_SERVER_URL");
```

Comment:
Actually, the change to externalize the ZIPKIN_SERVER_URL is on my queue. If you wouldn't mind, I'll send them as a separate PR.